### PR TITLE
Add doc for porting plugins

### PR DESCRIPTION
**Pull request by jjw24**

### CODE REVIEW

File: port-plugins.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Porting Wox/PowerToys Run Plugins
+
+Note:
```
Comment (Zeroto521) [inline port-plugins.md:3]: `Note:` -> `## Note` level 2 title

File: port-plugins.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- When porting, please keep the author's commit history
+- Flow Launcher targets .Net Core 3.1, so plugins should also be upgraded to keep the continuity of future developments
+
+Steps:
```
Comment (Zeroto521) [inline port-plugins.md:7]: `Steps:` -> `## Steps` level 2 title

**Review by Zeroto521 — CHANGES_REQUESTED** [review]
using `#` title mark replace `:`

**Comment by jjw24** [comment]
done, please take another look @Zeroto521

File: port-plugins.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Porting Wox/PowerToys Run Plugins
+
+## Note:
```
Comment (Zeroto521) [inline port-plugins.md:3]: image

It is better don't add colon `:`, it doesn't look good.

**Review by Zeroto521 — CHANGES_REQUESTED** [review]

**Comment by jjw24** [comment]
Done.

**Review by Zeroto521 — APPROVED** [review]

**Pull request merged**